protos/fpc/key_dist.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright IBM Corp. All Rights Reserved.
+// Copyright 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package key_distribution;
+option go_package = "github.com/hyperledger-labs/fabric-private-chaincode/internal/protos";
+
+message CCKeyRegistrationMessage {
+    bytes cc_params_hash = 1;       // defines the context as SHA2 of cc_params
+    bytes chaincode_ek   = 2;       // public chaincode encryption key
+    bytes enclave_id     = 3;       // creator of this message
+}
+
+message SignedCCKeyRegistrationMessage {
+    bytes msg_bytes = 1;            // public chaincode encryption key
+    bytes signature = 2;            // signature of the message creator (enclave)
+}
+
+message ExportMessage {
+    bytes cc_params_hash = 1;       // defines the context as SHA2 of cc_params
+    bytes chaincode_ek   = 2;       // public chaincode encryption key
+    bytes cckeys_enc     = 3;       // chaincode keys encrypted for the receiver
+    bytes enclave2_vk    = 4;       // receiver of this message
+    bytes enclave_vk     = 5;       // creator of this message
```

Comment:
The name doesn't really reflect the role, maybe better encode role also in name?
```suggestion
    bytes receiver_enclave_vk    = 4;       // receiver of this message
    bytes sender_enclave_vk     = 5;       // sender (creator) of this message
```